src/common/config_opts.h

```diff
@@ -396,7 +396,8 @@ OPTION(client_metadata, OPT_STR, "")
 OPTION(client_acl_type, OPT_STR, "")
 OPTION(client_permissions, OPT_BOOL, true)
 OPTION(client_dirsize_rbytes, OPT_BOOL, true)
-
+// consume by ceph_volume_client, use rados namespace to isolate cephFS volume instead of using pool
+OPTION(cephFSClient_data_isolate_by_namespace, OPT_BOOL, true)
```

Comment:
I don't think we need to make it optional: CephFSVolumeClient was only released with Jewel, so it does not have to work with any ceph versions that don't have the namespace support.